### PR TITLE
ban model, test cases, and help updates

### PR DESCRIPTION
**Pull request by TitanVJ**
## Description
fixes length of mod field in ban record model. updates to test cases for ban cog, and update to  convertbans examples in help.json

## PR Checklist

These are the things you need to ensure are covered in your PR, otherwise the CODEOWNERS will not approve your PR, not matter how much you ping them to do so on the Discord  
  
 1. The description in the PR is a fair representation of what the PR is about.
 1. The PR is fixing one thing and one thing only.
 1. Logging. if you have N variables initialzed/used in your function, you should print all of them out to the log using logging module at least once or have a good reason why you arent.
 1. If your PR is doing something like adding a new line or removing a new line, CODEOWNERS reserve the right to ask that you undo that change unless it was for a specific reason.
 1. If you are adding a new command....document. Document the following things on help.json and the README.md
    1. The purpose of the command
    1. If the argument is called with any arguments.
       1. If it is called with any arguments, please either provide a good enough explanation of the arg that a user can tell what it will do before using the command. adding an example of how to call it with the args is not necessary but good practice.
 1. If you are making a new Class of commands, add the class to bot.json following the convention already there.
 1. Evidence of Testing. This one needs to be completed after the PR is opened. At that point, you will go on the channel on the CSSS Wall-E Test Server that was automatically created when the PR was opened and then test the [following functionality](https://github.com/CSSS/wall_e/blob/master/documentation/Working_on_Bot/Test_Cases.md). Once you had done so, you can leave a comment on the PR stating that you had done the necessary testing.
 1. Please provide ways to test whatever you just modified on the bot in the [Test Cases section](https://github.com/CSSS/wall_e/blob/master/documentation/Working_on_Bot/Test_Cases.md) so that future PRs can be tested to ensure they dont break your code when merging to master

### CODE REVIEW

**Comment by modernNeo** [comment]
I know it's not technically part of this PR but can you change
https://github.com/TitanVJ/wall_e/blob/ee1fdd3893bd17023775d3c4533c2722fb553dc8/wall_e/src/WalleModels/models.py#L45-L50

and

https://github.com/TitanVJ/wall_e/blob/ee1fdd3893bd17023775d3c4533c2722fb553dc8/wall_e/src/resources/cogs/ban.py#L43-L44

to provide a better human-readable output. Cause 
```
022-09-11 20:24:52 = INFO - [Ban load()] loaded the following banned users: [80343821417721856, 80363843754131456, 128660664217501696, 131998558911070208, 159985870458322944, 165825416064532481, 167789170847252480, 167806482530697216, 184559004175826944, 19268095
```
is not very informative.

Maybe change it to something like
```
022-09-11 20:24:52 = INFO - [Ban load()] loaded the following banned users: [<discordUsername>(discordID),...
```
?

**Comment by TitanVJ** [comment]
> I know it's not technically part of this PR but can you change https://github.com/TitanVJ/wall_e/blob/ee1fdd3893bd17023775d3c4533c2722fb553dc8/wall_e/src/WalleModels/models.py#L45-L50
> 
> and
> 
> https://github.com/TitanVJ/wall_e/blob/ee1fdd3893bd17023775d3c4533c2722fb553dc8/wall_e/src/resources/cogs/ban.py#L43-L44
> 
> to provide a better human-readable output. Cause
> 
> ```
> 022-09-11 20:24:52 = INFO - [Ban load()] loaded the following banned users: [80343821417721856, 80363843754131456, 128660664217501696, 131998558911070208, 159985870458322944, 165825416064532481, 167789170847252480, 167806482530697216, 184559004175826944, 19268095
> ```
> 
> is not very informative.
> 
> Maybe change it to something like
> 
> ```
> 022-09-11 20:24:52 = INFO - [Ban load()] loaded the following banned users: [<discordUsername>(discordID),...
> ```
> 
> ?

i dont think its worth adding the names. i think it'll just be better if it prints a count of banned users loaded  instead of the ids. your thoughts?

**Comment by modernNeo** [comment]
> 
> i dont think its worth adding the names. i think it'll just be better if it prints a count of banned users loaded instead of the ids. your thoughts?

@TitanVJ 

I mean, I think my way is more informative, but considering that the output is really only relevant to the Minions, if ya'll are OK with just a count of banned users, then sure 😄

File: wall_e/src/resources/cogs/ban.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ async def load(self):
         # read in ban_list of banned users
         logger.info('[Ban load()] loading ban list from the database')
         self.ban_list = await BanRecords.get_all_active_ban_user_ids()
-        logger.info(f"[Ban load()] loaded the following banned users: {self.ban_list}")
+        logger.info(f"[Ban load()] loaded {len(self.ban_list)} banned users from database")
```
Comment (modernNeo) [inline wall_e/src/resources/cogs/ban.py:44]: this is a better idea: https://docs.djangoproject.com/en/2.2/ref/models/querysets/#count

**Review by modernNeo — CHANGES_REQUESTED** [review]
l

File: wall_e/src/resources/cogs/ban.py (hunk 1 above)
Comment (TitanVJ) [reply wall_e/src/resources/cogs/ban.py]: done

**Comment by modernNeo** [comment]
this ready for review?

**Comment by TitanVJ** [comment]
@modernNeo yes

**Review by modernNeo — APPROVED** [review]

**Pull request merged**